test/code/providers/runas_provider/runasprovider_test.cpp

```diff
@@ -564,7 +601,7 @@ class SCXRunAsProviderTest : public CPPUNIT_NS::TestFixture
             ofs << "exit 0" << std::endl;
         }
 
-        int err = system("chmod u+x testChRoot/setup.sh && bash -c ./testChRoot/setup.sh");
+        err = system("chmod u+x testChRoot/setup.sh && bash -c ./testChRoot/setup.sh");
```

Comment:
Ditto here, see above. We always strive to leave code in as good or better condition to how you find it. Given that you need to change this anyway, mind as well update to SCXProcess::Run() for much better error output.